stablehlo/dialect/TypeInference.cpp

```diff
@@ -3753,6 +3755,7 @@ LogicalResult verifyReduceWindowOp(
   }
   bool allInputsUnranked = (rankedInputIdx == -1);
   Block& block = body.front();
+  // reduce_window_c3, reduce_window_c13, reduce_window_i2
```

Comment:
As per https://github.com/openxla/stablehlo/pull/1280#issuecomment-1476730861
we have a missing check for i2. We can fix it is reduce op to be percolated in other ops.

Reply:
Let's leave this work to #704 as this requires some non-trivial changes.